recipes/pastrami/meta.yaml

```diff
@@ -0,0 +1,140 @@
+{% set name = "pastrami" %}
+{% set version = "0.9.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/healthdisparities/pastrami/archive/v{{ version }}.tar.gz"
+  sha256: 4457ad6cda7a2384a1b22109caf3edbff191b3e0a8749f779525276424b4c55c
+
+build:
+  noarch: python
+  number: '0'
+  string: py_0
```

Comment:
Why do you set this? It should not be necessary.

Reply:
Done